tests/foreman/cli/test_contentview.py

```diff
@@ -1106,6 +1108,42 @@ def test_positive_create_composite_with_component_ids(self):
             'the input values',
         )
 
+    @tier2
+    @skip_if_bug_open('bugzilla', 1487265)
+    @run_only_on('sat')
+    def test_negative_create_composite_with_component_ids(self):
+        """Try create a composite content view with a component_ids option
+            contains ids from same component
```

Comment:
Attempt to create a composite content view with a component_ids option which ids are from the same content view